src/masternode-payments.cpp

```diff
@@ -610,14 +610,21 @@ bool CMasternodePayments::GetBlockTxOuts(int nBlockHeight, CAmount blockReward,
             return false;
         }
 
-        if (dmnPayee->nOperatorReward == 0 || dmnPayee->pdmnState->scriptOperatorPayout == CScript()) {
-            voutMasternodePaymentsRet.emplace_back(masternodeReward, dmnPayee->pdmnState->scriptPayout);
-        } else {
-            CAmount operatorReward = (masternodeReward * dmnPayee->nOperatorReward) / 10000;
+        CAmount operatorReward = 0;;
+        if (dmnPayee->nOperatorReward != 0 && dmnPayee->pdmnState->scriptOperatorPayout != CScript()) {
+            // This calculation might eventually turn out to result in 0 even if an operator reward percentage is given
```

Comment:
Fixing punctuation for multiline comments
```suggestion
            // This calculation might eventually turn out to result in 0 even if an operator reward percentage is given.
```